types/splashy/splashy-tests.ts

```diff
@@ -0,0 +1,18 @@
+import splashy from 'splashy';
```

Comment:
```suggestion
import splashy = require('splashy');
```
IMO this is a CommonJS, until 3.9 we need above:
https://devblogs.microsoft.com/typescript/announcing-typescript-3-9-rc/#cjs-auto-imports